packages/@aws-cdk/aws-stepfunctions-tasks/test/lambda/invoke.test.ts

```diff
@@ -235,6 +235,18 @@ describe('LambdaInvoke', () => {
     }));
   });
 
+  test('Invoke lambda with payloadResponseOnly with invalid payload', () => {
+    // WHEN
+    const payload = sfn.TaskInput.fromText('invalid-payload');
```

Comment:
nit: can we be specific with the naming here and call it a `literal string` rather than invalid payload
It helps to map it to the methods exposed in `TaskInput`